### ISSUE TITLE
params validation as array of objects

### ISSUE THREAD

**Issue by narmontas** [majorleaguesoccer/respectify#11]
I am trying to get array of objects validated, but it seems it is not capable of it.

```
params: {
  destinations: {
    dataTypes: [ 'array', 'object' ],
    required: true,
    params: {
      price: {
        dataTypes: 'number',
        required: true
      }
    }
  }
}
```

I receive error when passing array of objects with price property.
```
{
  "code": "MissingParameter",
  "message": "Param `destinations.price` required",
  "parameter": {
    "name": "price",
    "paramType": "querystring",
    "dataTypes": [
      "number"
    ]
  }
}
```

**Comment by sorensen** [comment]
Well thats not good, I'll look into this.

**Comment by sorensen** [comment]
@narmontas I've published v2.2.0 which should fix this issue.

**Comment by narmontas** [comment]
Thank you @sorensen. It works now.
